lab-enrique/app/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "angulartest",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "node_modules/webpack/bin/webpack.js",
+    "watch": "webpack-dev-server --inline --hot"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
```

Comment:
ERROR in ./entry.js
Module not found: Error: Can't resolve 'style-loader' in '/Users/tommytnez/team401/rico/21-angular-webpack/lab-enrique/app'
 @ ./entry.js 3:0-28
 @ multi (webpack)-dev-server/client?http://localhost:8080 webpack/hot/dev-server ./entry.js

ERROR in ./entry.js
Module not found: Error: Can't resolve 'style-loader' in '/Users/tommytnez/team401/rico/21-angular-webpack/lab-enrique/app'
 @ ./entry.js 4:0-27
 @ multi (webpack)-dev-server/client?http://localhost:8080 webpack/hot/dev-server ./entry.js


these are the errors I am getting. check your package.json then resubmit